holmes/main.py

```diff
@@ -99,6 +99,11 @@ def init_logging(verbose = False):
     "-v",
     help="Verbose output",
 )
+opt_echo_request: bool = typer.Option(
+    True,
```

Comment:
Im not sure i would keep this as true even though this was the default behavior until now

Reply:
merging for now and open to changing later! thanks.